src/whylogs/io/local_dataset.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
this file should be `local_image_dataset` - it's tightly coupled with PIL logic and imports

Reply:
For now, I will extend it to other file formats.